docs/src/pages/theming/responsive/react.mdx

```diff
@@ -3,7 +3,7 @@ import { Example } from '@/components/Example';
 
 ## Example
 
-Import any of our primitives and use either the object or array syntax to changes styles responsively. Resize your browser window window to see the styles change.
+Simply import any of our primitives and use either the object or array syntax to changes styles responsively. Resize your browser window window to see the styles change.
```

Comment:
I try avoid words like 'simply' and 'clearly' because it might not actually be simple for some people and it doesn't add to the content. https://css-tricks.com/words-avoid-educational-writing/#3-simply

Reply:
Agreed, I was getting a build error around `import` and had changed this as a test. The problem ended up being elsewhere and forgot to change it back, so thanks for catching it. Nothing is every "simple"!!!!